README.md

```diff
@@ -0,0 +1,26 @@
+# Mobile & Hybrid App Analytics Accelerator
+
+This accelerator guides the user through setting up Snowplow tracking in mobile hybrid apps.
+It also introduces the Snowplow dbt mobile data model.
+
+## Installation
+
+Recursively update the git submodules:
+
+```
+git submodule update --init --recursive
+```
+
+Build the Hugo app:
+
+```
+./scripts/build.sh
+```
+
+## Usage
+
+Run an HTTP server inside the `public` folder. For example, using Python:
+
+```
+python3 -m http.server --directory public
```

Comment:
I'm a big fan of `npx http-server public` but maybe thats just the node in me speaking.

I wonder if we should pop in some `hugo server` docs?

Either way, the advanced analytics repo could do with this too!

Reply:
I couldn't resist a bit of shell scripting and updated the build script to be able to start a Hugo server directly using:

```
./scripts/build.sh serve
```

If it's okay, we can also add this to the web accelerator.

Reply:
Nice! Would be great to see this on the web one too.

If I'm being super nitpicky, it should probably be `run.sh` or something now, `build.sh build` seems weird.